### ISSUE TITLE
Unable to use Acala Bridge using latest Brave browser

### ISSUE THREAD

**Issue by ltfschoen** [AcalaNetwork/Acala#2710]
My Acala Address: 5F2QSwPHobet1pLFTEw6mkcT8bwZdEMmHfBGYPkiK7Pgxdex

**Describe the bug**

**Expected Behavior**

It should allow me to bridge tokens to Acala

**Current Behavior**

Acala Bridge appears unresponsive https://apps.acala.network/bridge.
Latest Brave Browser console gives error 
```
main.75c3f1f0.js:2 2024-02-22 10:04:26             VEC: Unable to decode on index 45 createType(Lookup0):: Invalid AccountId provided, expected 32 bytes, found 20

main.75c3f1f0.js:2 2024-02-22 10:04:26        RPC-CORE: subscribeStorage(keys?: Vec<StorageKey>): StorageChangeSet:: createType(Vec<StorageKey>):: createType(Lookup0):: Invalid AccountId provided, expected 32 bytes, found 20

Uncaught Error: createType(Vec<StorageKey>):: createType(Lookup0):: Invalid AccountId provided, expected 32 bytes, found 20
```

**Steps to Reproduce**

1. Go to https://apps.acala.network/bridge
2. Choose Polkadot to Acala
3. Choose to sent to same destination account as the origin account

**Additional context**

 - Browser: Brave lastest [Version 1.62.165 Chromium: 121.0.6167.184 (Official Build) (arm64)](https://brave.com/latest/)
  - OS: macOS 14.1.1

**Comment by xlc** [comment]
what extensions do you have?

**Comment by ltfschoen** [comment]
Polkadot.js, Metamask, Talisman, Argent X, Keplr.

Maybe it could also be something to do with this? https://github.com/polkadot-js/extension/issues/1250#issuecomment-1931926244

**Comment by xlc** [comment]
Yeah most likely you have evm account and the dapp wasn't handling it correctly. Will get someone to take a look.
